changelog/index.dd

```diff
@@ -2,7 +2,7 @@ Ddoc
 
 $(SCRIPT
     var redir = window.location.href.replace(
-        /\/changelog\/index.html#(?:2\.|new2_)(\d+(?:\.\d+)?)$(DOLLAR)/,
+        /\\/changelog\\/index.html#(?:2\\.|new2_)(\\d+(?:\\.\\d+)?)$(DOLLAR)/,
         "/changelog/2.$(DOLLAR)1.html"
```

Comment:
Wow, using inline js with regex inside ddoc files looks like a bad idea. All JS code should be defined in `*.js` files and included automatically included inline if really necessary. That said, this out of the scope of your PR, so I'm fine with leaving it like this.